test/cell_test.rb

```diff
@@ -59,9 +59,37 @@ def test_it_can_render
     cell.fire_upon
 
     assert_equal "H", cell.render
+    cell.fire_upon
+    cell.fire_upon
+    assert_equal "X", cell.render
   end
 
-  #write tests for H and X
+  def test_it_can_render_empty_hit_sunk
```

Comment:
all this seems to be working but I won't feel 100% confident until we get the game set up and display the board mid-game!